Workbooks/Cloudflare.json

```diff
@@ -0,0 +1,981 @@
+          "size": 0,
+          "title": "Top Threat User Agents",
+          "timeContext": {
+            "durationMs": 0
+          },
+          "timeContextFromParameter": "TimeRange",
+          "queryType": 0,
+          "resourceType": "microsoft.operationalinsights/workspaces",
+          "visualization": "table",
+          "tileSettings": {
+            "titleContent": {
+              "columnMatch": "title",
+              "formatter": 1
+            },
+            "leftContent": {
+              "columnMatch": "Count",
+              "formatter": 12,
+              "formatOptions": {
+                "palette": "auto"
+              },
+              "numberFormat": {
+                "unit": 17,
+                "options": {
+                  "maximumSignificantDigits": 3,
+                  "maximumFractionDigits": 2
+                }
+              }
+            },
+            "showBorder": false,
+            "sortCriteriaField": "Count",
+            "sortOrderField": 2,
+            "size": "auto"
+          }
+        },
+        "conditionalVisibility": {
+          "parameterName": "tab",
+          "comparison": "isEqualTo",
+          "value": "cloudflare_security_overview"
+        },
+        "customWidth": "33",
+        "name": "Top Threat User Agents - Copy"
+      },
+      {
+        "type": 3,
+        "content": {
+          "version": "KqlItem/1.0",
+          "query": "let 5xx = Cloudflare_CL \n| where tostring(EdgeResponseStatus_d) startswith \"5\"\n| summarize Count=count()\n| extend title=\"5xx Errors (Edge)\";\n\nlet 4xx = Cloudflare_CL \n| where tostring(EdgeResponseStatus_d) startswith \"4\"\n| summarize Count=count()\n| extend title=\"4xx Errors (Edge)\";\n\nlet 3xx = Cloudflare_CL \n| where tostring(EdgeResponseStatus_d) startswith \"3\"\n| summarize Count=count()\n| extend title=\"3xx Errors (Edge)\";\n\nlet result_table = union 5xx, 4xx, 3xx; \nresult_table \n| sort by Count\n\n",
```

Comment:
EdgeResponseStatus_d should be EdgeResponseStatus_s, please correct it.

Reply:
EdgeResponseStatus_d column is of number type.
![image](https://user-images.githubusercontent.com/76435946/111989837-f887ec00-8b1a-11eb-99d9-b9afe8a8395a.png)
![image](https://user-images.githubusercontent.com/76435946/111989955-19e8d800-8b1b-11eb-90c2-d11876ff94db.png)